forch/faucet_state_collector.py

```diff
@@ -375,18 +375,18 @@ def _augment_mac_urls(self, url_base, switch_data):
                 mac_data['url'] = f"{url_base}/?list_hosts?eth_src={mac}"
 
     @_pre_check()
-    def get_switch_state(self, switch, port, url_base=None):
+    def get_switch_state(self, switch, port, gauge_metrics, url_base=None):
```

Comment:
shouldn't this just be 'metrics' at this point?  Why/how is this specific to gauge here?

Reply:
There is already (faucet) metric used in the decorating function of this function. Just wanted to make it clear they are not the same thing..

Reply:
can you be more specific, I don't know what "the decorating function of this function" is...

Reply:
I was talking about "restore_states_from_metrics" but it seems that it is only used in the "process_*" functions... I think I will change "gauge_metrics" to just "metrics" here.